bundler/lib/dependabot/bundler/helpers.rb

```diff
@@ -20,10 +20,8 @@ def self.bundler_version(lockfile)
 
         if (matches = lockfile.content&.match(BUNDLER_MAJOR_VERSION_REGEX))
           matches[:version].to_i >= 2 ? V2 : V1
```

Comment:
I'm curious, under this new regime, what do we do if we think it is v1 generated lockfile? Do we blow up or do we try to update it using v2?

Reply:
When version is lower then 2 that's mean we are not supporting it. we are raising unsupported error by catching this version. Instead we expect customer to update their bundler version since switch from bundler v1 to bundler v2 is not smooth if there is library that doesn't supported by both. Here is explanation when v1 is specified in the repository. 

https://github.com/github/dependabot-updates/issues/6972#issuecomment-2377975253